sunbird-platform-core/common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -442,6 +442,9 @@
     String TO_ACCOUNT_ID_MISSING = "To Account id is mandatory.";
     String FROM_ACCOUNT_ID_NOT_EXISTS = "From Account id not exists";
     String PARAM_NOT_MATCH="%s-NOT-MATCH";
+    String CERT_ID_MISSING = "Cert id is mandatory.";
```

Comment:
use {0} is mandatory , instead of all mandatory field.
{0} will be replace with actual value.

Reply:
done as suggested